internal/provider/trust_provider_models.go

```diff
@@ -29,61 +29,85 @@ type trustProvidersDataSourceModel struct {
 }
 
 type trustProviderAzureMetadataModel struct {
-	Sku            types.String `tfsdk:"sku"`
-	VMID           types.String `tfsdk:"vm_id"`
-	SubscriptionID types.String `tfsdk:"subscription_id"`
+	Sku             types.String   `tfsdk:"sku"`
+	Skus            []types.String `tfsdk:"skus"`
```

Comment:
This is all still relatively new to me... but if I am understanding this correctly, one can define Sku and Skus. Won't it be confusing that you can define a single element and an array at the same time. And they will be combined? Or am I totally off here?

Reply:
@mgavrilov-aembit Thank you for the reminder - yes, the TF Provider validation should prevent that - update to add that validation in like it exists for GitLab incoming.